keps/sig-cloud-provider/2395-removing-in-tree-cloud-providers/kep.yaml

```diff
@@ -22,15 +22,15 @@ approvers:
   - "@liggit"
 editor: TBD
 creation-date: 2018-12-18
-last-updated: 2019-04-11
+last-updated: 2023-04-20
 status: implementable
 
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.21"
+latest-milestone: "v1.27"
 
-stage: alpha
+stage: beta
```

Comment:
Does it mean that it went beta in 1.27 without being tracked via release team, PRR approval, etc.?

Reply:
OK - queued for review.

In the meantime, can you please add `milestones` field:
https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/kep.yaml#L33
to make it clear that Beta is still ahead of us :)